bcrs-ui/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "hello-world",
+  "name": "vue-frontEnd-Framework",
```

Comment:
This should actually be this project's name (though it should be generic for the template project).

Reply:
Good point! I will rename this, and i'm really glad you mentioned the template as i will have to PR these same updates to the Entities repo where we are keeping a snapshot. Moving forward after this i think it makes sense to personalize to the app ( like you suggested! )